tests/core/editor/manual/customconfigrace.html

```diff
@@ -0,0 +1,24 @@
+<h2>Editor 1</h2>
+<div id="editor1"></div>
+<h2>Editor 2</h2>
+<div id="editor2"></div>
+<h2>Editor 3</h2>
+<div id="editor3"></div>
+
+<script>
+	var assetsPath = '%BASE_PATH%core/editor/_assets/',
+		config1 = assetsPath + 'raceconfig1.js',
+		config2 = assetsPath + 'raceconfig3.js';
```

Comment:
I guess you chose `raceconfig3.js` instead of `raceconfig2.js` here to make the difference in editors width as clear as possible, but at first glance it looks like a typo. I checked how does it look with `raceconfig2.js` and I think it will still be easy to verify 👍